engine/src/main/java/org/terasology/rendering/dag/nodes/FinalPostProcessingNode.java

```diff
@@ -121,7 +121,7 @@ public FinalPostProcessingNode(Context context) {
         addDesiredStateChange(new SetInputTextureFromFbo(texId++, TONE_MAPPING_FBO_URI, ColorTexture, displayResolutionDependentFBOs, POST_MATERIAL_URN, "texScene"));
         addDesiredStateChange(new SetInputTextureFromFbo(texId++, lastUpdatedGBuffer, DepthStencilTexture, displayResolutionDependentFBOs, POST_MATERIAL_URN, "texDepth"));
         setBlurTexStateChage = new SetInputTextureFromFbo(texId++, SECOND_LATE_BLUR_FBO_URI, ColorTexture, displayResolutionDependentFBOs, POST_MATERIAL_URN, "texBlur");
-        // addDesiredStateChange(new SetInputTexture3D(texId++, "engine:colorGradingLut1", POST_MATERIAL_URN, "texColorGradingLut"));
+        addDesiredStateChange(new SetInputTexture3D(texId++, "engine:colorGradingLut1", POST_MATERIAL_URN, "texColorGradingLut"));
         // TODO: use GLSL noise functions to have a completely shader-based film grain.
```

Comment:
Let's rewrite this as: 

`// TODO: evaluate the possibility to use GPU-based noise algorithms instead of CPU-generated textures.`